aiowebostv/webos_client.py

```diff
@@ -502,15 +386,16 @@ def clear_state_update_callbacks(self) -> None:
 
     async def do_state_update_callbacks(self) -> None:
         """Call user state update callback."""
-        if callbacks := {callback(self) for callback in self.state_update_callbacks}:
-            await asyncio.gather(*callbacks)
+        if not self.state_update_callbacks or not self.do_state_update:
+            return
```

Comment:
This condition was used in all of the occurrences we call `do_state_update_callbacks` and it makes more sense to do it here instead of every occurrence of `do_state_update_callbacks`